docs/experiment-analysis/data-pipeline.md

```diff
@@ -44,4 +44,17 @@ Funnel metrics require their own pipeline because of the complicated logic that
 
 1. **Incremental funnel metric events**: We incrementally join assignment data to the event sources for your funnel metrics.
 2. **Funnel metrics data frame**: We generate a data frame for each subject and day for each funnel metric.
-3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
\ No newline at end of file
+3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
+
+## Privacy
+
+As we’ve detailed, Eppo doesn’t export individual data from your warehouse. We actively minimize what information we hold.
+
+* **Fact creation**: We query small sample of tables to help you set-up and make sure the column names and types match. We do not send those results beyond your analysts’ browser nor do we retain them; we just store the configuration. We can anonymize that step by only showing the data type, not the exact value. We do not need access, and encourage you *not* to give our user agent access to any personally identifiable information (PII), just unique identifiers that you have generated.
+* **Incremental pipelines**: Our pipelines query daily aggregate of assignments and facts, we only store those on your warehouse. We only keep a copy of the text of the queries that we generate and run (where table and column names appear, but no individual information). We do not keep individual outputs for any daily query. We add them up into incremental tables and store those in your warehouse too.
+* **Auditability**: All of our queries and the resulting tables are fully auditable by your data engineers. All of our process rely on a system user whose access you control and can reboke at any time. Your engineers send us the token for that account directly to our secure enclave, no Eppo employee has access to those. You can update that token on your own if needed.
```

Comment:
"Our queries" instead of "All of our queries"
"Our processes" instead of "All of our process"
"service account" instead of "system user"
"revoke" misspelled as "reboke". Correct.
Your engineers configure the service account connection via our web app and we store the credentials in Google Secret Manager.  You can update credentials on your own if needed. **Don't include the no Eppo employee part...a few limited admins do have access to the secrets**